docs/content/middlewares/http/ipallowlist.md

```diff
@@ -193,3 +193,7 @@ http:
     [http.middlewares.test-ipallowlist.ipAllowList.ipStrategy]
       excludedIPs = ["127.0.0.1/32", "192.168.1.7"]
 ```
+
+### `rejectStatusCode`
+
+The `rejectStatusCode` option sets HTTP status code for refused requests. If not set, the default is 403 (Forbidden).
```

Comment:
I think the option should be also documented for every provider, as the other options (like above).

Reply:
Done